Products/ZenModel/migrate/addOOMParams.py

```diff
@@ -0,0 +1,48 @@
+##############################################################################
+#
+# Copyright (C) Zenoss, Inc. 2019, all rights reserved.
+#
+# This content is made available according to terms specified in
+# License.zenoss under the directory where your Zenoss product is installed.
+#
+##############################################################################
+
+import logging
+import Migrate
+import servicemigration as sm
+
+from Products.ZenModel.ZMigrateVersion import SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION
+from Products.ZenUtils.controlplane.client import getCCVersion
+
+log = logging.getLogger("zen.migrate")
+sm.require("1.1.13")
+
+
+class AddOOMParams(Migrate.Step):
+    "Add OOMKillDisable and OOMScoreAdj parameters to db services"
+
+    version = Migrate.Version(SCHEMA_MAJOR, SCHEMA_MINOR, SCHEMA_REVISION)
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        service_names = ['mariadb-model', 'redis', 'RegionServer', 'ZooKeeper',
+                         'HMaster', 'Impact', 'Solr', 'mariadb-events']
+        services = filter(lambda s: s.name in service_names, ctx.services)
+        log.info("Found %i services" % len(services))
+
+        cc_version = getCCVersion().split('.')
+        if int(cc_version[1]) < 6 and int(cc_version[2]) < 5:
```

Comment:
This `if` statement fails for _all_ 1.6.x versions, not just 1.6.5 and later, e.g.
```
>>> 6 < 6 and 4 < 6
False
```
An alternative way to handle this is to use `parse_version` from `pkg_resources`, e.g.
```
from pkg_resources import parse_version

cc_version = parse_version(getCCVersion())
if cc_version < parse_version("1.6.5"):
    ...
```
A little cleaner and simpler to read.